mods/xpanes/init.lua

```diff
@@ -129,7 +129,6 @@ function xpanes.register_pane(name, def)
 		groups = groups,
 		drop = "xpanes:" .. name .. "_flat",
 		sounds = def.sounds,
-		use_texture_alpha = def.use_texture_alpha or false,
```

Comment:
Should be left in.

Reply:
this is so weird, I actually had not even removed this in my original commit. I am using the current pull for this branch, but it seems to have differences from my 0.4.16 install

is it possible some changes were made since 0.4.16 release, that I am now reverting by "fixing" all of these lines?

Reply:
These lines were the lines changed in the previous commit to xpanes, you may have somehow reverted the last commit.